week3/homework/src/todos.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
there are 2 todo.json why do you think that is ? 😄

Reply:
Ow... I think I committed the list by mistake

Reply:
can you remove it ?